en/phones.ftl

```diff
@@ -133,3 +133,66 @@ phone-banner-cta-user = Upgrade now
 #   $nr_texts (number) - the number of texts one can send and receive per month
 phone-banner-float-limits = { $nr_calls } call minutes + { $nr_texts } texts
 phone-banner-float-replies = Reply to texts privately
+
+## Frequently Asked Questions about Phone plans
```

Comment:
These can all be seen at https://relay.firefox.com/faq/ if you're in the US or Canada.